README.md

```diff
@@ -166,28 +166,57 @@ Formatter Name          | Description | Since
 
 ## Development
 
-To develop tslint-microsoft-contrib simply clone the repository, install dependencies and run grunt:
+To develop `tslint-microsoft-contrib` simply clone the repository, install dependencies and run `npm test`:
 
     git clone git@github.com:Microsoft/tslint-microsoft-contrib.git --config core.autocrlf=input --config core.eol=lf
     cd tslint-microsoft-contrib
     npm install
-    grunt all
-    grunt create-rule --rule-name=no-something-or-other
+    npm test
+
+You can create new rule from template with `create-rule` script:
+
+    npm run create-rule -- --rule-name=no-something-or-other
+
+> NOTE: `--` is required before script arguments.
+
+This script will create file for rule implementation (inside `src`) as well as folder with rule tests (inside `test`).
+
+More information about writing rule tests can be found in [TSLint doccumentation](https://palantir.github.io/tslint/develop/testing-rules/)
+
 
 ### Debug code
 
-If command fails because of file access permissions, prefix it with sudo.
+**VS Code**
+
+This repo provides pre-configured launch tasks for VS Code debugging rules and tests.
+
+* Pick `Run Mocha Tests` to debug rules which have Mocha unit tests inside `src/tests` folder.
 
-    npm install -g node-inspector
+* Pick `Run TSLint Tests` to debug rules that have tests in TSLint format inside `tests` folder.
 
-Then run:
+**Node.js**
 
-    node-debug grunt mochaTest
+Starting from Node.js v6.3 has built-in debugger that can be used with Chrom DevTools. For earlier versions use [`node-inspector`](https://www.npmjs.com/package/node-inspector) package.
 
-The `node-debug` command will load Node Inspector in your default browser (works in Chrome and Opera only).
+To debug rules that have Mocha tests use:
+
+    node --inspect-brk ./node_modules/mocha/bin/_mocha --no-timeouts --colors "dist/src/tests/**/*.js"
```

Comment:
Nit: would you mind using <code>```</code> formatting here? GitHub's renderer thinks the `_` before `mocha` is an italic block starter.

<pre><code>```shell
node ...
```
</code></pre>

![image](https://user-images.githubusercontent.com/3335181/47511794-f4235000-d848-11e8-93a7-8fdf25c36d25.png)

Reply:
I usually do this, but decided to follow already used style. Will change.